test/mock/MockBlastPoints.sol

```diff
@@ -2,5 +2,10 @@
 pragma solidity ^0.8.0;
 
 contract MockBlastPoints {
-    function configurePointsOperator(address operator) external {}
+    mapping(address _contract => address operator) public contractOperators;
+    event LogAddress(address);
```

Comment:
can you remove this event, it's only for logging

Reply:
I can't find the commit I did for this but it's fixed now